src/main/java/Storage.java

```diff
@@ -0,0 +1,90 @@
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    private static String filePath;
+
+    public Storage() {}
+
+    public Storage(String filePath) {
+        Storage.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() throws FileNotFoundException {
+        File file = new File(filePath);
+        ArrayList<Task> list = new ArrayList<>();
+        Scanner sc = new Scanner(file);
+        while (sc.hasNext()) {
+            String line = sc.nextLine(); // eg. T,0,read book
+            String[] taskDetails = line.split(",");
+            String taskType = taskDetails[0];
+            int booleanInt = Integer.parseInt(taskDetails[1]);
```

Comment:
Just a matter of preference, but I think its better to change this into boolean directly here